pkg/controller/expand_and_recover.go

```diff
@@ -191,7 +191,7 @@ func (ctrl *resizeController) expandAndRecover(pvc *v1.PersistentVolumeClaim, pv
 func (ctrl *resizeController) markForSlowRetry(pvcKey string, resizeStatus v1.ClaimResourceStatus) {
 	if resizeStatus == v1.PersistentVolumeClaimControllerResizeInfeasible ||
 		resizeStatus == v1.PersistentVolumeClaimNodeResizeInfeasible {
-		ctrl.slowSet.Add(pvcKey)
+		ctrl.slowSet.Add(pvcKey, resizeStatus)
```

Comment:
This is the only Add call and it's always called with `PersistentVolumeClaimNodeResizeInfeasible`, so why all the code around multiple states?

Reply:
No it can be called with either `NodeResizeInfeasible` or `ControllerResizeInfeasible`. The idea is to not accidentally clear `NodeResizeInfeasible` errors if controller expansion is successful or fails with a non-infeasible error.